app/konnect/getting-started/configure-mesh.md

```diff
@@ -16,14 +16,14 @@ For the purposes of this guide, you'll use {{site.konnect_short_name}} to create
 
 ## Prerequisites
 
-* [A Kubernetes cluster with a load balancer](https://kubernetes.io/docs/setup/)
+* [A Kubernetes cluster with a load balancer](https://kubernetes.io/docs/setup/) <!--what are we load balancing? Which pods and why?-->
```

Comment:
@johnharris85 When I was setting this up, I was a bit confused by the load balancing recommendation. Should we provide additional guidance about what they should be load balancing and why?

Reply:
This should probably link to https://kubernetes.io/docs/concepts/services-networking/service/#publishing-services-service-types and say "A Kubernetes cluster with a load balancer service capabilities".

I don't think we need to provide more guidance than that.